airflow/providers/amazon/aws/hooks/batch_client.py

```diff
@@ -103,6 +103,7 @@ def submit_job(
         parameters: dict,
         containerOverrides: dict,
         ecsPropertiesOverride: dict,
+        eksPropertiesOverride: dict,
```

Comment:
Please make it optional to avoid adding breaking change.

Reply:
you mean Optional[dict] ? then, other parameters change same?

Reply:
Actually, from #39903, it seems it is not a breaking change ... You can ignore this comment :) Sorry about that